spyder/widgets/projects/explorer.py

```diff
@@ -67,7 +67,7 @@ def toggle_hscrollbar(self, checked):
         if PYQT5:
             self.header().setSectionResizeMode(QHeaderView.ResizeToContents)
         else:
-            self.header().setResizeMode(QHeaderView.ResizeToContents)

```

Comment:
We need a try/except here to support older `qtpy` versions. We could rewrite this `if` block as

```python
try:
    self.header().setSectionResizeMode(QHeaderView.ResizeToContents)
except:
    self.header().setResizeMode(QHeaderView.ResizeToContents)
```

and remove the `PYQT5` constant